test/db/cmd/cmd_http_post

```diff
@@ -0,0 +1,11 @@
+NAME=http-post-cmd
+FILE==
+CMDS=<<EOF
+!python3 -m pip install requests

```

Comment:
No, this is not the way to do it. Please update these files instead:
- `.builds/*`
- `.github/workflows/*`
- `.appveyor.yml`
- `.travis.yml`